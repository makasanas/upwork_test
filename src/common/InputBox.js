import React, { Component } from 'react';
import '../assets/css/input_box.css';

class InputBox extends Component{
    constructor(props){
        super(props)
    }
    render(){
        return(
            <div className="input-box">
               <div className="itemName">{this.props.title ? this.props.title:''}</div>
                <input type="text" name={this.props.name} value={this.props.value} onChange={(e)=>this.props.onUpdate(e,"inputbox")}/>
            </div>
        );
    }
}
export default InputBox;