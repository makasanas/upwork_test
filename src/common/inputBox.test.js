import React from 'react';
import InputBox from './inputBox';

test('render a input commone components', () => {
  const wrapper = shallow(
      <InputBox  store={createStoreWithMiddleware(reducers)}/>
  );
  expect(wrapper).toMatchSnapshot();
});


