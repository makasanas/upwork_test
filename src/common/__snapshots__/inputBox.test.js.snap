// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a input commone components 1`] = `
<div
  className="input-box"
>
  <div
    className="itemName"
  />
  <input
    onChange={[Function]}
    type="text"
  />
</div>
`;
